src/openrct2/ride/TrackData.cpp

```diff
@@ -12597,7 +12627,8 @@ namespace OpenRCT2::TrackMetaData
         /* TrackElemType::DiagBlockBrakes */
         { 4, { kDiagBlockBrakesSeq0, kDiagBlockBrakesSeq1, kDiagBlockBrakesSeq2, kDiagBlockBrakesSeq3 } },
         /* TrackElemType::InclinedBrakes */
-        { 1, { kDown25Seq0 } }
+        { 1, { kDown25Seq0 } },
+        /* TrackElemType::DiagBooster */ { 4, { kDiagBoosterSeq0, kDiagBoosterSeq1, kDiagBoosterSeq2, kDiagBoosterSeq3 } },
```

Comment:
I thought so, but I wasn't certain since you did it for the diag brakes. I don't really understand what it's saying here. I'll change it to the same sequence as DiagFlat, then.

Reply:
Applied now, tested, works indeed.